R/impact_central.R

```diff
@@ -370,18 +373,25 @@ impact_by_birth_year <- function(baseline_burden, focal_burden) {
   assert_has_columns(focal_burden,
```

Comment:
Thinking about moving to calling this with a `tbl_dbi` [object](https://dbplyr.tidyverse.org/reference/tbl.src_dbi.html) I am not sure how this will work - it might force the data to be fetched.

Reply:
Sorry, not sure I understand this comment

Reply:
For the science team if we add a function for them to calculate impact with data from the database this will be called with a dbpyr `tbl_dbi` object. I expect that the `assert_has_columns` will force the data to be fetched.

Reply:
we can probably write a way of doing that through dbplyr I imagine?